client/src/modules/cohort/index.jsx

```diff
@@ -27,18 +33,62 @@ class Cohort extends PureComponent {
     fetchListMetaData(id);
   }
 
+  showUpdateForm = () => {
+    this.setState({ updateFormVisibility: true });
+  };
+
+  hideUpdateForm = () => {
+    this.setState({ updateFormVisibility: false });
+  };
+
+  updateCohortHandler = cohortId => (name, description) => {
+    const { cohortDetails, updateCohort } = this.props;
+    const {
+      description: previousDescription,
+      name: previousName
+    } = cohortDetails;
+    const dataToUpdate = {};
+    name ? (dataToUpdate.name = name) : null;
+    description ? (dataToUpdate.description = description) : null;
```

Comment:
please use ternary operator only if return value is used
move these checks and `const dataToUpdate` inside the if in line 54 - we do not need to do those if they are the same